se_denseslam/src/bfusion/alloc_impl.hpp

```diff
@@ -73,13 +73,13 @@ size_t buildOctantList(HashType* allocationList, size_t reserved,
   unsigned int voxelCount;
 #endif
 
-  int x, y;
+  int y;
   const Eigen::Vector3f camera = pose.topRightCorner<3, 1>();
   voxelCount = 0;
 #pragma omp parallel for \
   private(y)
```

Comment:
This syntax is necessary in C, not C++. Can you amend the loops to declare the iteration variables in the for-loop body? Also, declaring private(y) would be unnecessary.